routes/ecom/modules/calculate.js

```diff
@@ -191,7 +191,8 @@ module.exports = appSdk => {
                 }
                 if (servicePkg.weight) {
                   shippingLine.package.weight = {
-                    value: parseFloat(servicePkg.weight)
+                    value: parseFloat(servicePkg.weight),
+                    unit: 'kg'
```

Comment:
```suggestion
                    unit: 'g'
```

Uai mas não seria `g` aqui então? Ou então você divide o valor por 1000...

Reply:
A resposta chega 0.6 kg, no caso o sistema está salvando 0.6 g

Reply:
Porque o default da api é g

Reply:
Como o talisson não colocou nada ai, já salva por padrão como g e ai quando vai enviar para gerar a etiqueta no melhor envio, vai com a dimensão correta, mas com o peso errado

Reply:
Ah entendi